firefox_puppeteer/tests/test_toolbars.py

```diff
@@ -135,3 +137,71 @@ def test_matching_text(self):
             self.assertTrue(len(all_matches) > 0)
             for match_fragment in all_matches:
                 self.assertIn(match_fragment, (input_text, input_text.upper()))
+
+
+class TestIdentityPopup(FirefoxTestCase):
+
+    def test_box(self):
+        self.assertIsNotNone(
+            self.browser.navbar.locationbar.identity_popup.box)
```

Comment:
And maybe you want to use a `setUp` method where you can set `self.identity_popup = self.browser.navbar.locationbar.identity_popup`, and to use it as shortcut in your tests.